modules/ocf_prometheus/manifests/alertmanager.pp

```diff
@@ -0,0 +1,40 @@
+class ocf_prometheus::alertmanager {
+  ocf::firewall::firewall46 {
+    '995 allow prometheus alertmanager to send on SMTP port':
+      opts   => {
+        chain       => 'PUPPET-OUTPUT',
+        proto       => 'tcp',
+        destination => ['anthrax'],
```

Comment:
we shouldn't hardcode the server's names if we can use the service CNAMEs instead

Reply:
i also don't think an array is necessary here?